src/utils/backendCrossNotebookRequest.ts

```diff
@@ -21,21 +23,29 @@ const backendCrossNotebookRequest = async <T>({
   )
     .toString()
     .split(":");
+
+  // START OF SETUP
+  // TODO - simplify this setup to import methods directly from package. 
+  // Don't need a full client with websocket
   const settings = {
     token,
     uuid,
   };
-  const { sendNotebookRequest, unload, listNotebooks } = setupSamePageClient({
-    renderOverlay: () => () => {},
+  setupRegistry({
```

Comment:
I think this is the brunt of the double render problem. We were setting up another websocket connection with the same notebook uuid, so the response was being sent to the wrong place